Specs/Core/EllipsoidRhumbLineSpec.js

```diff
@@ -0,0 +1,581 @@
+        var ninetyPointZeroTwoDegrees = 90.02 * oneDegree;
+
+        var rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, eightyNineDegrees, distance, ellipsoid);
+        var rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, eightyNinePointNineDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, ninetyDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, ninetyPointOneDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, ninetyPointZeroTwoDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+    });
+
+    it('tests spheroid close to 90 degrees', function() {
+        var ellipsoid = Ellipsoid.WGS84;
+        var fifteenDegrees = Math.PI / 12;
+        var initial = new Cartographic(fifteenDegrees, fifteenDegrees);
+        var distance = ellipsoid.maximumRadius * fifteenDegrees;
+
+        var oneDegree = CesiumMath.RADIANS_PER_DEGREE;
+        var eightyNineDegrees = 89 * oneDegree;
+        var eightyNinePointNineDegrees = 89.9 * oneDegree;
+        var ninetyDegrees = 90 * oneDegree;
+        var ninetyPointOneDegrees = 90.1 * oneDegree;
+        var ninetyPointZeroTwoDegrees = 90.02 * oneDegree;
+
+        var rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, eightyNineDegrees, distance, ellipsoid);
+        var rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON3);
```

Comment:
Why the change in epsilon vs. the sphere case? Might be worth documenting if it's deliberate due to the precision of the implementation.